components/notifications-service2/cmd/notifications-service/commands/serve.go

```diff
@@ -46,5 +46,4 @@ func runServeCommand(cmd *cobra.Command, args []string) error {
 		fmt.Println(string(s))
 		time.Sleep(5 * time.Second)
 	}
-	return nil
```

Comment:
Don't we need to always return something?

Reply:
I guess go know's it's unreachable since the loop is never broken. `govet` failed on it 🤷‍♂️